2024-04-12 10:38:46,145 INFO    MainThread:280362 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-12 10:38:46,146 INFO    MainThread:280362 [wandb_setup.py:_flush():76] Configure stats pid to 280362
2024-04-12 10:38:46,146 INFO    MainThread:280362 [wandb_setup.py:_flush():76] Loading settings from /home/jeongseokoh/.config/wandb/settings
2024-04-12 10:38:46,146 INFO    MainThread:280362 [wandb_setup.py:_flush():76] Loading settings from /data2/jeongseokoh/jeongseokoh/bnn/wandb/settings
2024-04-12 10:38:46,146 INFO    MainThread:280362 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-12 10:38:46,146 INFO    MainThread:280362 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-04-12 10:38:46,146 INFO    MainThread:280362 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-12 10:38:46,146 INFO    MainThread:280362 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-12 10:38:46,146 INFO    MainThread:280362 [wandb_init.py:_log_setup():521] Logging user logs to /data2/jeongseokoh/jeongseokoh/bnn/wandb/run-20240412_103846-t4f3c4xf/logs/debug.log
2024-04-12 10:38:46,147 INFO    MainThread:280362 [wandb_init.py:_log_setup():522] Logging internal logs to /data2/jeongseokoh/jeongseokoh/bnn/wandb/run-20240412_103846-t4f3c4xf/logs/debug-internal.log
2024-04-12 10:38:46,147 INFO    MainThread:280362 [wandb_init.py:_jupyter_setup():467] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f02a4100860>
2024-04-12 10:38:46,148 INFO    MainThread:280362 [wandb_init.py:init():561] calling init triggers
2024-04-12 10:38:46,148 INFO    MainThread:280362 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.01, 'architecture': 'Scratch', 'dataset': 'de,en', 'epochs': 10}
2024-04-12 10:38:46,148 INFO    MainThread:280362 [wandb_init.py:init():586] re-initializing run, found existing run on stack: vwrgmo8w
2024-04-12 10:38:46,150 INFO    MainThread:280362 [wandb_run.py:_finish():2064] finishing run luke0112/Sparse_plane/vwrgmo8w
2024-04-12 10:38:46,150 INFO    MainThread:280362 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-04-12 10:38:46,150 INFO    MainThread:280362 [wandb_run.py:_restore():2287] restore
2024-04-12 10:38:46,151 INFO    MainThread:280362 [wandb_run.py:_restore():2293] restore done
2024-04-12 10:38:50,483 INFO    MainThread:280362 [wandb_run.py:_footer_history_summary_info():3936] rendering history
2024-04-12 10:38:50,484 INFO    MainThread:280362 [wandb_run.py:_footer_history_summary_info():3968] rendering summary
2024-04-12 10:38:50,494 INFO    MainThread:280362 [wandb_run.py:_footer_sync_info():3895] logging synced files
2024-04-12 10:38:50,541 INFO    MainThread:280362 [wandb_init.py:init():611] starting backend
2024-04-12 10:38:50,541 INFO    MainThread:280362 [wandb_init.py:init():615] setting up manager
2024-04-12 10:38:50,544 INFO    MainThread:280362 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-04-12 10:38:50,544 INFO    MainThread:280362 [wandb_init.py:init():623] backend started and connected
2024-04-12 10:38:50,553 INFO    MainThread:280362 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-04-12 10:38:50,553 INFO    MainThread:280362 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-04-12 10:38:50,553 INFO    MainThread:280362 [wandb_init.py:init():715] updated telemetry
2024-04-12 10:38:50,567 INFO    MainThread:280362 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-04-12 10:38:51,961 INFO    MainThread:280362 [wandb_run.py:_on_init():2357] communicating current version
2024-04-12 10:38:52,033 INFO    MainThread:280362 [wandb_run.py:_on_init():2366] got version response 
2024-04-12 10:38:52,033 INFO    MainThread:280362 [wandb_init.py:init():799] starting run threads in backend
2024-04-12 10:38:56,303 INFO    MainThread:280362 [wandb_run.py:_console_start():2335] atexit reg
2024-04-12 10:38:56,303 INFO    MainThread:280362 [wandb_run.py:_redirect():2190] redirect: wrap_raw
2024-04-12 10:38:56,303 INFO    MainThread:280362 [wandb_run.py:_redirect():2255] Wrapping output streams.
2024-04-12 10:38:56,303 INFO    MainThread:280362 [wandb_run.py:_redirect():2280] Redirects installed.
2024-04-12 10:38:56,304 INFO    MainThread:280362 [wandb_init.py:init():842] run started, returning control to user process
2024-04-12 10:42:51,925 INFO    MainThread:280362 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-12 10:42:51,925 INFO    MainThread:280362 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-12 10:44:02,985 INFO    MainThread:280362 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-12 10:44:03,007 INFO    MainThread:280362 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-12 10:44:03,007 INFO    MainThread:280362 [wandb_setup.py:_flush():76] Configure stats pid to 280362
2024-04-12 10:44:03,007 INFO    MainThread:280362 [wandb_setup.py:_flush():76] Loading settings from /home/jeongseokoh/.config/wandb/settings
2024-04-12 10:44:03,007 INFO    MainThread:280362 [wandb_setup.py:_flush():76] Loading settings from /data2/jeongseokoh/jeongseokoh/bnn/wandb/settings
2024-04-12 10:44:03,007 INFO    MainThread:280362 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-12 10:44:03,007 INFO    MainThread:280362 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-04-12 10:44:03,008 INFO    MainThread:280362 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-12 10:44:03,008 INFO    MainThread:280362 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-12 10:44:03,008 INFO    MainThread:280362 [wandb_init.py:_log_setup():521] Logging user logs to /data2/jeongseokoh/jeongseokoh/bnn/wandb/run-20240412_104402-e0ng78re/logs/debug.log
2024-04-12 10:44:03,008 INFO    MainThread:280362 [wandb_init.py:_log_setup():522] Logging internal logs to /data2/jeongseokoh/jeongseokoh/bnn/wandb/run-20240412_104402-e0ng78re/logs/debug-internal.log
2024-04-12 10:44:03,008 INFO    MainThread:280362 [wandb_init.py:init():561] calling init triggers
2024-04-12 10:44:03,009 INFO    MainThread:280362 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.01, 'architecture': 'Scratch', 'dataset': 'de,en', 'epochs': 10}
2024-04-12 10:44:03,009 INFO    MainThread:280362 [wandb_init.py:init():586] re-initializing run, found existing run on stack: t4f3c4xf
2024-04-12 10:44:03,011 INFO    MainThread:280362 [wandb_run.py:_finish():2064] finishing run luke0112/Sparse_plane/t4f3c4xf
2024-04-12 10:44:03,011 INFO    MainThread:280362 [jupyter.py:save_history():445] not saving jupyter history
2024-04-12 10:44:03,011 INFO    MainThread:280362 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-12 10:44:03,011 INFO    MainThread:280362 [wandb_init.py:_jupyter_teardown():449] cleaning up jupyter logic
2024-04-12 10:44:03,011 INFO    MainThread:280362 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-04-12 10:44:03,012 INFO    MainThread:280362 [wandb_run.py:_restore():2287] restore
2024-04-12 10:44:03,012 INFO    MainThread:280362 [wandb_run.py:_restore():2293] restore done
2024-04-12 10:44:09,996 INFO    MainThread:280362 [wandb_run.py:_footer_history_summary_info():3936] rendering history
2024-04-12 10:44:09,997 INFO    MainThread:280362 [wandb_run.py:_footer_history_summary_info():3968] rendering summary
2024-04-12 10:44:10,008 INFO    MainThread:280362 [wandb_run.py:_footer_sync_info():3895] logging synced files
